tools/attrd_updater.c

```diff
@@ -45,6 +45,8 @@ static struct crm_option long_options[] = {
     {"-spacer-",1, 0, '-', "\nCommands:"},
     {"update",  1, 0, 'U', "Update the attribute's value in attrd.  If this causes the value to change, it will also be updated in the cluster configuration"},
 #ifdef HAVE_ATOMIC_ATTRD
+    {"update-all",   1, 0, 'E', "Update the attribute's value, and the time to wait (dampening) in attrd.  If this causes the value to change, it will also be updated in the cluster configuration"},
```

Comment:
I'm thinking something like this:
{"update-both",   1, 0, 'B', "Update the attribute's value and time to wait (dampening) in attrd. If this causes the value or dampening to change, the attribute will also be written to the cluster configuration, so be aware that repeatedly changing the dampening reduces its effectiveness."}